apps/penxle.com/src/routes/publish/Header.svelte

```diff
@@ -241,250 +240,203 @@
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'HORROR')}>공포성</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'GAMBLING')}>사행성</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'CRIME')}>약물/범죄</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'PHOBIA')}>정신질환/공포증</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'TRAUMA')}>PTSD/트라우마</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'INSULT')}>부적절한 언어</ToggleButton>
+            <ToggleButton size="lg" on:change={(e) => checkContentFilter(e, 'OTHER')}>기타</ToggleButton>
+          </div>
+        {/if}
 
-      <div class="flex gap-1.5">
-        <input
-          id="allowTagContribution"
-          class="square-6"
-          checked={receiveTagContribution}
-          type="radio"
-          value={receiveTagContribution}
-          on:input={() => (receiveTagContribution = true)}
-        />
-        <label class="grow body-15-sb" for="allowTagContribution">허용</label>
-      </div>
-      <div class="flex gap-1.5">
-        <input
-          id="notAllowTagContribution"
-          class="square-6"
-          checked={!receiveTagContribution}
-          type="radio"
-          value={receiveTagContribution}
-          on:input={() => (receiveTagContribution = false)}
-        />
-        <label class="grow body-15-sb" for="notAllowTagContribution">비허용</label>
-      </div>
-    </fieldset>
-
-    <div class="body-16-b flex gap-6">
-      <div class="flex items-center gap-1">
-        <span>게시글 피드백</span>
-        <Tooltip message="오래 머문 구간 표시, 가장 많은 밑줄을 친 구간" placement="top">
-          <i class="i-lc-help-circle square-4 text-secondary" />
-        </Tooltip>
-      </div>
+        <p class="text-secondary">세부 설정</p>
 
-      <div class="flex gap-1.5">
-        <input
-          id="receiveFeedback"
-          class="square-6"
-          checked={receiveFeedback}
-          type="radio"
-          on:input={() => (receiveFeedback = true)}
-        />
-        <label class="grow body-15-sb" for="receiveFeedback">활성화</label>
-      </div>
-      <div class="flex gap-1.5">
-        <input
-          id="notReceiveFeedback"
-          class="square-6"
-          checked={!receiveFeedback}
-          type="radio"
-          on:input={() => (receiveFeedback = false)}
-        />
-        <label class="grow body-15-sb" for="notReceiveFeedback">비활성화</label>
-      </div>
-    </div>
+        <div class="flex items-center justify-between">
+          <div>
+            <p class="body-16-b">게시물 태그 수정 및 등록</p>
+            <p class="body-15-m text-secondary">다른 독자들이 게시물의 태그를 수정할 수 있어요</p>
+          </div>
+          <Switch bind:checked={receiveTagContribution} />
+        </div>
```

Comment:
P3: 체크박스처럼 레이블 영역을 클릭하면 스위치가 토글되게 부탁드립니다!

```suggestion
        <label class="flex items-center justify-between">
          <div>
            <p class="body-16-b">게시물 태그 수정 및 등록</p>
            <p class="body-15-m text-secondary">다른 독자들이 게시물의 태그를 수정할 수 있어요</p>
          </div>
          <Switch bind:checked={receiveTagContribution} />
        </label>
```

Switch 컴포넌트를 슬롯으로 레이블을 넣을 수 있는 구조로 개선하면 아래처럼 더 축약해서 가능할 듯:


```svelte
<Switch bind:checked={receiveTagContribution}>
  <p class="body-16-b">게시물 태그 수정 및 등록</p>
  <p class="body-15-m text-secondary">다른 독자들이 게시물의 태그를 수정할 수 있어요</p>
</Switch>
```